docs/release/release-process.md

```diff
@@ -18,6 +18,15 @@ level sequential overview for how to run the release process.
        tests in the release branch
     1. **tag**: Run the tag pipeline to tag the release branch with an official semver
     1. **build/publish**: Run build pipeline to publish build with official semver
+1. **release template stacks**: Run the release process for each
+   **template stack** that we maintain. Note that the process for
+   template stacks is slightly different from the stack release process.
+    1. **test** Test builds from the release branch, fix any critical
+       bugs that are found
+    1. **version**: Update all version information in the docs, as
+       appropriate
+    1. **tag**: Run the tag pipeline to tag the release branch with an official semver
```

Comment:
i think you mentioned the template stacks aren't using release branches? if so, we could remove release branch from this sentence

on the bigger topic of release branches, what's the thinking/justification for releasing directly from master? simplicity and streamlining?

if we want to do a patch release for the last release after master has moved on, will we create a release branch that starts at the commit we released from master? (i.e. retroactively create the release branch)

Reply:
looks like you partially answer this below with:

> For Template Stacks, we do not use release branches unless we need to
make a patch release. In the future we may need a more robust branching
strategy, but for now we are not using branches because it is simpler.

but still curious about doing a patch later on, if we need to retroactively create a release branch 🤔

Reply:
I'm now noticing that there is already a patch release section to this document which tells the reader to go to the instructions in each other section, so I will think a little bit more about whether we can add anything here to help.

Reply:
I've added some notes about creating patch releases for template stacks